### ISSUE TITLE
Binance fetchClosedOrders() retrieves data from Binance but return array is missing nearly all data from method call

### ISSUE THREAD

**Issue by rxall** [ccxt/ccxt#12981]
**Version Info**
- OS: Ubuntu 20.04.3
- Programming Language version: PHP 8.1.4 
- CCXT version: 1.80.7

**Description**
This was working with CCXT version 1.42.23, after updating to the latest (/version above) it ceased working. The data is still returned from the ```allOrders``` endpoint but doesn't appear to be returned correctly from the ```$exchange->fetchClosedOrders``` call. 

**Code to Reproduce**
```
<?php
date_default_timezone_set('UTC');

$exchange_id = 'binance';
$exchange_class = '\\ccxt\\' . $exchange_id;
$exchange = new $exchange(array(
    'apiKey': '[removed]',
    'secret': '[removed]',
));

$exchange->verbose = true; 
$trades = $exchange->fetchClosedOrders("LTC/GBP");
print_r($trades);
```

**Output**
```
Array
(
    [0] => fetch Request:
    [1] => binance
    [2] => GET
    [3] => https://api.binance.com/api/v3/allOrders?timestamp=[removed]&symbol=LTCGBP&recvWindow=5000&signature=[removed]
    [4] => RequestHeaders:
    [5] => Array
        (
            [X-MBX-APIKEY] => [removed]
        )

    [6] => RequestBody:
    [7] => 
)
Array
(
    [0] => fetch Response:
    [1] => binance
    [2] => GET
    [3] => https://api.binance.com/api/v3/allOrders?timestamp=[removed]&symbol=LTCGBP&recvWindow=5000&signature=[removed]
    [4] => 200
    [5] => 
    [6] => ResponseHeaders:
    [7] => Array
        (
        [removed]
        )

    [8] => ResponseBody:
    [9] => [{"symbol":"LTCGBP","orderId",[removed],"orderListId":-1,"clientOrderId":"[removed]","price":"88.59000000","origQty":"[removed]","executedQty":"13.15524000","cummulativeQuoteQty":"1165.42271160","status":"FILLED","timeInForce":"GTC","type":"LIMIT","side":"BUY","stopPrice":"0.00000000","icebergQty":"0.00000000","time":[removed],"updateTime":[removed],"isWorking":true,"origQuoteOrderQty":"0.00000000"}, {"symbol":"LTCGBP","orderId":[removed],"orderListId":-1,"clientOrderId":"x-[removed]","price":"131.13000000","origQty":"9.97500000","executedQty":"9.97500000","cummulativeQuoteQty":"1308.02175000","status":"FILLED","timeInForce":"GTC","type":"LIMIT","side":"BUY","stopPrice":"0.00000000","icebergQty":"0.00000000","time":[removed],"updateTime":[removed],"isWorking":true,"origQuoteOrderQty":"0.00000000"}] 
)
Array
(
    [0] => closed
    [1] => closed
)
```

Potentially related to https://github.com/ccxt/ccxt/issues/12925

**Comment by kroitor** [comment]
@rxall i've tried to reproduce it with CCXT 1.80.52, but it works for me... I'm on PHP 7.4.26.

What is the the runtime version of CCXT in your script?

```PHP
<?php
date_default_timezone_set('UTC');

echo 'CCXT v' . \ccxt\Exchange::VERSION . "\n"; // <<<<<<<<<<<<<<< ADD THIS

$exchange_id = 'binance';
$exchange_class = '\\ccxt\\' . $exchange_id;
$exchange = new $exchange(array(
    'apiKey': '[removed]',
    'secret': '[removed]',
));

$exchange->verbose = true; 
$orders = $exchange->fetch_closed_orders("LTC/GBP");
print_r($orders);
```

**Comment by rxall** [comment]
@kroitor thanks for checking it out 

```
CCXT v1.80.52
``` 

I also just tried it with ```PHP 7.4.29``` and I'm still getting the same issue. However, using the older version of CCXT (```v1.42.1```) I get the correct result with both versions of PHP.

Any other ideas? Just to confirm I'm expecting output similar to a) from the ```print_r``` but I'm getting b)

a) 
```
    [0] => Array
        (
            [info] => Array
                (
                    [symbol] => LTCGBP
                    [orderId] => [removed]
                    [orderListId] => -1
                    [clientOrderId] => [removed]
                    [price] => 88.59000000
                    [origQty] => 13.15524000
                    [executedQty] => 13.15524000
                    [cummulativeQuoteQty] => 1165.42271160
                    [status] => FILLED
                    [timeInForce] => GTC
                    [type] => LIMIT
                    [side] => BUY
                    [stopPrice] => 0.00000000
                    [icebergQty] => 0.00000000
                    [time] =>  [removed]
                    [updateTime] =>  [removed]
                    [isWorking] => 1
                    [origQuoteOrderQty] => 0.00000000
                )

            [id] => [removed]
            [clientOrderId] => [removed]
            [timestamp] => [removed]
            [datetime] => [removed]
            [lastTradeTimestamp] => 
            [symbol] => LTC/GBP
            [type] => limit
            [timeInForce] => GTC
            [postOnly] => 
            [side] => buy
            [price] => 88.59
            [stopPrice] => 0
            [amount] => 13.15524
            [cost] => 1165.4227116
            [average] => 88.59
            [filled] => 13.15524
            [remaining] => 0
            [status] => closed
            [fee] => 
            [trades] => 
        )

    [1] => Array
    
    (example cut here)
```

b)
```
Array
(
    [0] => closed
    [1] => closed
    [2] => closed
    [3] => closed
    [4] => closed
    [5] => closed
    [6] => closed
    [7] => closed
    [8] => closed
    [9] => closed
    [10] => closed
)
```

**Comment by kroitor** [comment]
Interestingly, i get the output (a) on my side:

<img width="554" alt="Screenshot 2022-04-23 at 17 09 06" src="https://user-images.githubusercontent.com/1294454/164909491-284a1ff8-075b-45c4-9644-bf8f1afb6db1.png">

@rxall Do you have the same result from fetch_orders as from fetch_closed_orders ?

**Comment by rxall** [comment]
@kroitor ```fetch_orders()``` returns correctly a filled array for me using both mentioned versions of CCXT

**Comment by kroitor** [comment]
@rxall ok, got it! Please hold on, it looks like a bug with the fetch_closed_orders filtering, we'll fix it shortly!

**Comment by kroitor** [comment]
@rxall thx for reporting it, I've pushed the fixes in CCXT 1.80.56. Please, let us know if you have this issue with the new version after upgrading. Appreciate your feedback!

**Comment by rxall** [comment]
🙏 all good, thanks for the fix
